core/kover/learning/experiment.py

```diff
@@ -451,14 +474,14 @@ def normal_raise(exception):
         best_model, \
         best_rule_importances, \
         best_predictor_equiv_rules = _bound_selection(dataset_file, split_name, model_type, p, max_rules,
-                                                      max_equiv_rules, bound_delta, bound_max_genome_size, n_cpu,
-                                                      random_generator, progress_callback, warning_callback,
+                                                      max_equiv_rules, rule_blacklist, bound_delta, bound_max_genome_size, 
```

Comment:
Would you mind modifying the calls to _bound_selection, _cross_validation and _full_train  to use keyword arguments? I think this would leave less rooms for errors.